SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -464,11 +479,12 @@ void UGlobalStateManager::AuthorityChanged(bool bWorkerAuthority, Worker_EntityI
 		{
 			SetCanBeginPlay(true);
 			BecomeAuthoritativeOverAllActors();
-			TriggerBeginPlay();
+			TryTriggerBeginPlay();
+		}
+		else
+		{
+			SetAcceptingPlayers(true);
```

Comment:
And then we're not doing an authority check for the component here.

Reply:
Shouldn't "if (bWorkerAuthority)" above do the job?

Reply:
That's checking a different component.

Reply:
I can add this line then as in TryTriggerBeginPlay?
`if (NetDriver->StaticComponentView->HasAuthority(GlobalStateManagerEntityId, SpatialConstants::DEPLOYMENT_MAP_COMPONENT_ID))`